tests/FSharp.Compiler.ComponentTests/Conformance/UnionTypes/E_LowercaseWhenRequireQualifiedAccess.fsx

```diff
@@ -0,0 +1,22 @@
+﻿// #Conformance #TypesAndModules #Unions 
+// This testcase verifies that lower-case discriminated union are not allowed
+//<Expects status="error"></Expects>
```

Comment:
Those are actually not required for newly written tests, they are there for the tests which were auto-migrated from the old framework.

Reply:
Cool I will remove them :)